Code/GraphMol/Fingerprints/MorganGenerator.h

```diff
@@ -108,9 +108,10 @@ template <typename OutputType>
 class RDKIT_FINGERPRINTS_EXPORT MorganArguments
     : public FingerprintArguments<OutputType> {
  public:
-  const bool df_includeChirality;
-  const bool df_onlyNonzeroInvariants;
-  const unsigned int d_radius;
+  const bool df_includeChirality = false;
+  const bool df_onlyNonzeroInvariants = false;
+  const unsigned int d_radius = 3;
+  const bool df_includeRedundantEnvironments = false;
```

Comment:
I guess these here such that some default values are present when the default constructor is invoked.

Reply:
> I guess these here such that some default values are present when the default constructor is invoked.

Yeah, and because I think it generally makes sense to initialize variables when they are declared